addons/payment/models/payment_transaction.py

```diff
@@ -488,6 +586,8 @@ def _handle_feedback_data(self, provider, data):
         tx = self._get_tx_from_feedback_data(provider, data)
         tx._process_feedback_data(data)
         tx._execute_callback()
+        if tx.operation == 'refund':
+            tx._cron_finalize_post_processing()
```

Comment:
For Adyen, the post-processing must happen after the reception of the notification, that's why it's called in `_handle_feedback_data` and not in `_send_refund_request` or any function called before the notification is received.
It'll probably be different for other providers, but they'll still use `_handle_feedback_data`, maybe in `_send_refund_request` as it's done for other methods. So, I made the change for the trigger, but I think it'd be better to let it there.

Reply:
After discussion, we agreed to create the trigger in Adyen's override of `_process_feedback_data` because:
1. Only Adyen currently implements refunds.
2. Forgetting to trigger the cron in another acquirer is not a big deal: the cron runs every 10 min anyway.
3. No clean and suitable spot was found in generic payment code.